src/api.jl

```diff
@@ -134,12 +134,12 @@ function infer(ra_range::Tuple{Float64, Float64},
     # Get indicies of entries in the  RA/Dec range of interest.
     entry_in_range = entry->((ra_range[1] < entry.pos[1] < ra_range[2]) &&
```

Comment:
Here's a picture.  Object 1 and object 2 both contribute flux to each
other's region of the sky, but neither shows up in the catalog of each
other's frames.  Consequently, they will not be listed in each other's
tile_sources.  Please correct me if I've misunderstood something.

On Tue, Mar 22, 2016 at 4:42 PM, Kyle Barbary <notifications@github.com>
wrote:

> In src/api.jl
> <https://github.com/jeff-regier/Celeste.jl/pull/174#discussion_r57089148>:
>
> > @@ -134,12 +134,12 @@ function infer(ra_range::Tuple{Float64, Float64},
> >      # Get indicies of entries in the  RA/Dec range of interest.
> >      entry_in_range = entry->((ra_range[1] < entry.pos[1] < ra_range[2]) &&
>
> What will happen if you choose an (RA, DEC) rectangle in
> query_overlapping_fieldids that contains the edge of a frame?
>
> That frame will be included.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/jeff-regier/Celeste.jl/pull/174/files/249556c8da20320ca889e1dfa8c6faef40973454#r57089148>
>